juriscraper/opinions/united_states/state/alaska.py

```diff
@@ -44,20 +49,22 @@ def _process_html(self) -> None:
             return
         for table in self.html.xpath("//table"):
             adate = table.xpath("./preceding-sibling::h5")[0].text_content()
-            if self.is_backscrape and not self.date_is_in_backscrape_range(
-                adate
+            if (
+                not self.date_is_in_range(adate)
+                and not self.test_mode_enabled()
             ):
                 logger.debug("Backscraper skipping %s", adate)
                 continue
 
             for row in table.xpath(".//tr"):
                 if row.text_content().strip():
-                    # skip rows without PDF links in first column
                     try:
                         url = get_row_column_links(row, 1)
                     except IndexError:
-                        continue
+                        url = ""
```

Comment:
we cant return an empty url that will just fail or break..  but also we can reconstruct the URL without the need of the secondary call. take a look at the old JSON data - the URL can be reconstructed with the data available on the page